src/components/panel/panel.js

```diff
@@ -0,0 +1,284 @@
+/**
+ * @ngdoc module
+ * @name material.components.panel
+ */
+angular
+    .module('material.components.panel', [
+      'material.core'
+    ])
+    .service('$mdPanel', MdPanelService);
+
+/**
+ * @ngdoc service
+ * @name $mdPanel
+ * @module material.components.panel
+ *
+ * @description
+ * `$mdPanel` is a general component that can be used for any panels on the screen,
+ * such as a tooltip, dialog, or custom popup.
```

Comment:
```
`$mdPanel` is a robust, low-level service for creating floating panels on the screen. It can be used to implement tooltips, dialogs, pop-ups, etc.
```